common/src/main/java/com/wynntils/wynn/model/CharacterManager.java

```diff
@@ -78,12 +82,25 @@ public static void onWorldStateChanged(WorldStateEvent e) {
 
         if (e.getNewState() == WorldStateManager.State.WORLD
                 && e.getOldState() != WorldStateManager.State.CHARACTER_SELECTION) {
-            McUtils.sendMessageToClient(new TextComponent(
-                            "Could not find your class type. Disable auto join (/toggle autojoin) and try again.")
-                    .withStyle(ChatFormatting.DARK_RED));
+            WynntilsMod.info("Scheduling character info query");
+            scanCharacterInfoPage();
         }
     }
 
+    private static void scanCharacterInfoPage() {
+        ScriptedContainerQuery query = ScriptedContainerQuery.builder("Character Info Query")
+                .useItemInHotbar(InventoryUtils.COMPASS_SLOT_NUM)
+                .matchTitle("^§.\\d+§. skill points remaining$")
```

Comment:
Odd, wouldn't expect this to be the title of the compass 🤔

Reply:
Yeah, it is odd, isn't it? But that is the actual title of the character info menu.

Reply:
Nvm, I was confused, it's the container's title not the item's